### DISCUSSION TITLE
"Error: Couldn't find manifest matching bottle checksum." for nss

### DISCUSSION THREAD

**Discussion by svewap** [Homebrew/discussions#2624] [Getting started] [answered]
Hello, does anyone know what to do here? Thank you for your help.

==> Upgrading 1 outdated package:
drud/ddev/ddev 1.18.0 -> 1.18.2
==> Downloading https://ghcr.io/v2/homebrew/core/readline/manifests/8.1.1
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/readline/blobs/sha256:8bb1e20aafd3dfc3076b62d8f55db867edd93cbfaeb44d1d03f1968fc51209d9
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:8bb1e20aafd3dfc3076b62d8f55db867edd93cbfaeb44d1d03f1968fc51209d9?se=2021-12-11T21%3A20%3A00Z&sig=w25adtnDDBj
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/nss/manifests/3.71
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/nss/manifests/3.71
######################################################################## 100.0%
Error: Couldn't find manifest matching bottle checksum.
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/home/user/.linuxbrew/Homebrew/Library/Homebrew/software_spec.rb:413:in `github_packages_manifest_resource_tab'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/software_spec.rb:366:in `fetch_tab'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/formula.rb:2389:in `fetch_bottle_tab'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/formula_installer.rb:1180:in `fetch'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/formula_installer.rb:701:in `fetch_dependency'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/formula_installer.rb:1168:in `block in fetch_dependencies'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/formula_installer.rb:1168:in `each'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/formula_installer.rb:1168:in `fetch_dependencies'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/formula_installer.rb:1175:in `fetch'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/upgrade.rb:71:in `block in upgrade_formulae'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/upgrade.rb:53:in `map'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/upgrade.rb:53:in `upgrade_formulae'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/cmd/upgrade.rb:192:in `upgrade_outdated_formulae'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/cmd/upgrade.rb:110:in `upgrade'
/home/user/.linuxbrew/Homebrew/Library/Homebrew/brew.rb:110:in `<main>'

**Comment by Bo98** [comment, marked as answer]
Your homebrew-core installation appears to be a couple months old. What does `brew config` output? Does `brew update` output any errors?

**Reply by svewap** [reply]
> HOMEBREW_VERSION: 3.3.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: dbec9de77ebc65a4845a5e672950219aa19b35b8
Last commit: 6 days ago
Core tap ORIGIN: https://github.com/Linuxbrew/homebrew-core
Core tap HEAD: 51b47956b0713413d134a533759a1873e53a3578
Core tap last commit: 7 weeks ago
Core tap branch: master
HOMEBREW_PREFIX: /home/user/.linuxbrew
HOMEBREW_REPOSITORY: /home/user/.linuxbrew/Homebrew
HOMEBREW_CELLAR: /home/user/.linuxbrew/Cellar
HOMEBREW_BROWSER: /usr/bin/google-chrome
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_DISPLAY: :0
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /home/user/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/bin/ruby
CPU: octa-core 64-bit zen
Clang: N/A
Git: 2.34.1 => /bin/git
Curl: 7.80.0 => /bin/curl
Kernel: Linux 5.15.5-1-default x86_64 GNU/Linux
OS: openSUSE Tumbleweed
Host glibc: 2.34
/usr/bin/gcc: 11.2.1
/usr/bin/ruby: 2.7.5
glibc: N/A
gcc@5: N/A
xorg: N/A

And:
>  brew update
Already up-to-date.

**Reply by Bo98** [reply]
Ah looks like you've found a bug. We've migrated from `Homebrew/linuxbrew-core` to `Homebrew/homebrew-core`, but you have the even older `Linuxbrew/homebrew-core` (which redirects to `Homebrew/linuxbrew-core`).

Try run `git -C "$(brew --repo homebrew/core)" remote set-url origin https://github.com/Homebrew/linuxbrew-core` and then do `brew update` again. It should trigger the migration properly this time.

**Reply by svewap** [reply]
Thank you very much!
